lib/headers.js

```diff
@@ -2,15 +2,18 @@
 var is = require('type-is');
 
 module.exports = function (app, options) {
+  var remotes = app.remotes();
```

Comment:
You have broken out remotes here, but up further you used app.remotes() on the same line.